README.md

```diff
@@ -10,6 +10,12 @@ CB-Log is the logger library for the Cloud-Barista Multi-Cloud Framework.
     - sol: $ rm -rf $GOPATH/src/go.etcd.io/etcd/vendor/google.golang.org/grpc
   
   B.  export CBLOG_ROOT=$GOPATH/src/github.com/cloud-barista/cb-log
+
+  - cf) if meet errors
+    - error msg: "go run sample.go 2020/10/13 22:42:13 error: open /src/github.com/cloud-barista/cb-log/conf/log_conf.yaml: no such file or directory exit status 1"
+    - sol: 
+      - 1) check there is $CBLOG_ROOT  (cd $CBLOG_ROOT shoud be equal to $HOME/go/src/github.com/cloud-barista/cb-log. It means that you can see conf folder when you type CD $CBLOG_ROOT)
```

Comment:
@yoonjiOh 

Trivial comments.
[AS-IS]
  CD $CBLOG_ROOT
[TO-BE]
  cd $CBLOG_ROOT